app/src/main/kotlin/compiler/App.kt

```diff
@@ -1,11 +1,56 @@
 package compiler
 
-fun main() {
-    // TODO:
-    // - create Reader for input source file
-    // - create Writer for output assembly file
-    // - call Compiler.process()
-    // - report any diagnostics
-    // - execute NASM on output assembly file
-    // - execute GCC or LD on resulting object file
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import compiler.diagnostics.CompilerDiagnostics
+import java.io.File
+import java.io.FileNotFoundException
+import java.io.FileReader
+import java.io.PrintWriter
+
+fun runCommand(command: String) {
+    println(command)
+    ProcessBuilder(*command.split(" ").toTypedArray())
+        .redirectOutput(ProcessBuilder.Redirect.INHERIT)
+        .redirectError(ProcessBuilder.Redirect.INHERIT)
+        .start()
+}
+
+class Cli : CliktCommand() {
+    val inputFile: String by argument("input file", help = "Input file")
+    val externFile: String? by option("-e", "--extern", help = "Extern file")
```

Comment:
I would name this argument as `stdlib` and (temporarily) assume the default value of `stdlib.o` in the current directory.